### ISSUE TITLE
Homework 1 Feedback 

### ISSUE THREAD

**Issue by Gchism94** [INFO526-DataViz/hw-01-meganhokama#2]
Dear Megan -- Below is the feedback for your assignment. Please review carefully, and stop by office hours if you have any questions about the feedback.

---

### Exercise 1: `[10 / 10 points]`

- Plot: `[6 / 6 points]`
- Narrative: `[4 / 4 points]`
- Feedback: Your plot looks great, good work! I would however like to see your insight into why a density plot was used here instead of other plot types. Additionally, you have some formatting issues with (`::: {.cell layout-align="center"}
```{.r .cell-code}`), and narrative text should be plain text outside of chunks. 

- [x] Resolved

### Exercise 2: `[16 / 20 points]`

a. `[8 / 8 points]`

  - Histogram: `[3 / 3]`
  - Boxplot: `[3/ 3]`
  - Narrative: `[2 / 2]`
  - Feedback: Your boxplot y-axis doesn't make sense, I suggest removing it entirely. I would also like a more thorough compare/contrast between the histogram and boxplot, such as what specifically each plot is telling you about the data. 
- [x] Resolved

b. `[4 / 5 points]`

  - Boxplot: `[2 / 3]`
  - Narrative: `[2 / 2]`
  - Feedback: Your narrative was good, but your fill color choice doesn't make sense to me. The color itself is very dark, so you can barely see the median line, but the same color for men and women prevents the viewer from distinguishing the two as easily. Lastly, I would like to see a cleaner y-axis label. 
- [ ] Resolved (no)

c. `[1 / 2 points]`

  - Feedback: You are correct, however you removed the legend both times `theme(legend.position="none")`. You also have some arguments that are not connected to the main `ggplot()` with a `+`. In the plot without redundancy, I suggest having different colors for each gender, but no legend. Again, the y-axis needs to be changed. 
- [ ] Resolved (no)

d. `[3 / 5 points]`

  - Time series plot: `[3 / 3]`
  - Narrative: `[0 / 2]`
  - Feedback: I don't see a narrative as to what the data is showing you. 

- [ ] Resolved (no)

### Exercise 3: `[20 / 20 points]`

a. `[2 / 2]`
  - Feedback: The code chunk was not formatted correctly. I agree with your narrative in principle, but I cannot see the plot to confirm. 
  - [x] Resolved 

b. `[2 / 2]`
  - Feedback: Please reproduce the plots so that I can directly confirm your narrative. You are correct however that the vertical facets are more appropriate. 
  - [x] Resolved 

c. `[16 / 16]` (2 points each)
  - Feedback: None of these were reproduced. Please reach out with questions. 
  - [x] Resolved 

### Exercise 4: `[10 / 10 points]`

- Plot: `[6 / 6 points]`
- Narrative: `[4 / 4 points]`
- Feedback: A tile plot is good when you have a limited number of axis ticks. This has literally 1000s of possible points, so a tile plot is not very effective. I suggest a distribution plot (e.g., histogram, density plot) compared across several facets. Please reach out with questions. 
  - [x] Resolved 
Additional feedback: I would have faceted the histograms, but this is a major improvement over the tile plot. 

### Exercise 5: `[20 / 20 points]`

Feedback: This is a pretty good attempt, but there is a lot of ink being used unnecessarily, and some labels are cut off in the final plot. 

  - [x] Resolved 

### Overall: `[15 / 20 points]`

- `[3 / 5]` - Doesn't require reproduction: .md file is in repo, figures show up in the .md file on GitHub, no egregious errors requiring reproducing the entire analysis to be able to follow it.
 Feedback: There were some reproduction issues that I had to address. You can get these points back after fixing the formatting issues. 
  - [ ] Resolved (no)
Additional feedback: I had to do several rounds of debugging. 
- `[1 / 3]` - Code style: Line breaks after `|>` and `+`, spaces around operators, spaces after commas, lines not too long, etc.
 Feedback: There are a few instances where ggplot layers are not connected to the main plot. 
  - [ ] Resolved (no)

- `[2 / 3]` - Code smell: Messy / unnecessarily complex code, difficult to follow, unaddressed warnings, etc.
 Feedback: There are a ton of messages and warnings that can be ignored in the chunk - `{r chunk name, warnings = FALSE, message = FALSE}`
  - [ ] Resolved (no)

- `[3 / 3]` - Quality and quantity of commits: No uninformative/nonsense text in commit messages, entire assignment not committed in a single commit.
- `[3 / 3]` - Reasonably sized figures.
- `[3 / 3]` - Font size, organization: No crazy large text for narrative, questions answered in order and identified, easy to follow.

## Late penalty

- [x] On time: No penalty
- [ ] Late, but same day (before midnight): -10 points
- [ ] Late, but next day: -20 points

**Comment by meganhokama** [comment]
Thank you so much for all of your very helpful feedback~ I tried my best in resolving some of the problems addressed but I wasn't able to get to all of them. I'm so sorry. 

Here is what I was able to address: 
Exercise 1: Added more justification to density plot. Additionally, I tried to address the format issue you brought up. 

Exercise 3: I added the ggplots for the grid and arranged them as well. Additionally, I was slightly confused with the producing the columns as they are already imaged in the ReadMe so I wasn't sure if I just needed to copy and paste that code you wrote in the ReadMe into the code. 

Exercise 4: Changed the plot to a histogram and adjusted the narrative accordingly.
